### PR TITLE
Remove async prefix

### PR DESCRIPTION
**Pull request by gregw**
https://github.com/GoogleCloudPlatform/jetty-runtime/issues/20
parameterized docker image tags and labels for ease of CI

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,7 +35,8 @@
     <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
     <jetty.version>9.3.8.v20160314</jetty.version>
     <appengine.api.version>1.9.40</appengine.api.version>
-    <docker.label.prefix>async-</docker.label.prefix>
+    <docker.tag>${jetty.version}</docker.tag>
```
Comment (meltsufin) [inline pom.xml]: I think the pattern we agreed on is "9.3-${build_id}", where the build_id is a timestamp of the build.

Comment (gregw) [inline pom.xml]: Ah yes - forgot that.  
So should we change the project version from 0.1.0-SNAPSHOT   to buildID-SNAPSHOT  and just use that?

Comment (meltsufin) [inline pom.xml]: I'm not really concerned about the Maven artifact version right now. I think we still need to figure out if we want to be publishing to Maven Central at all.

Comment (gregw) [inline pom.xml]: Then I'd recommend making the artifact version the build ID.  we then get auto increment whenever a release is done - no matter where that is released to.

Comment (meltsufin) [inline pom.xml]: We're trying to be consistent with other language runtimes, and they seem to be adopting the timestamp `YYYY-MM-DD_HH_MM` as the most specific docker version tag.

**Pull request merged**